.github/workflows/certification.yml

```diff
@@ -201,6 +203,7 @@ jobs:
       run: |
         go mod download
         go install gotest.tools/gotestsum@latest
+        go install github.com/axw/gocov/gocov@985a38bc91b08298f246092a1b242b258a0331cf
```

Comment:
Should we use a version v1.1.0 instead of the commit hash?

Reply:
I want to gather comments here and have an understanding that what should be the correct/better way.
With version, it becomes more explicit to anyone to understand that what version with what features, a particular repo is being used.
But, a particular Notice line in `https://github.com/Ilshidur/action-discord` caught my attention, which also seemed to make sense: `NOTICE : for stability purposes, it is recommended to use the action with an explicit commit SHA-1`

Reply:
It looks like they are asking to prefer a fixed version over `master`, as mentioned in the example. Version will be easier to maintain and upgrade, unless we specifically need to pin to a SHA-1 hash.